cas-server-core/src/main/java/org/jasig/cas/monitor/AbstractPoolMonitor.java

```diff
@@ -123,7 +123,7 @@ public PoolStatus observe() {
          *
          * @param monitor the monitor
          */
-        public Validator(final AbstractPoolMonitor monitor) {
+        Validator(final AbstractPoolMonitor monitor) {
```

Comment:
Maybe :) At the same time, less stuff to write. But I understand, we have been so used to creating "public" classes and using that keyword without hesitation that now removing it feels a bit strange.